load_test.py

```diff
@@ -0,0 +1,211 @@
+import sys
+import os.path
+sys.path.append('../')
+from pathlib import Path
+import os, random, uuid, requests, argparse
+from app.main import ENDPOINTS 
+from env_config.env_config import EnvConfig
+from utils import to_oauth_request_form
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+BASE_PATH = Path(__file__).parent.resolve()
+
+# Fixed User Credentials
+user_credentials = {
+    "password": "Test1@123",
+    "password_confirm": "Test1@123",
+    "validation_link": "tabpfn"
+}
+
+def parse_args():
+    """
+    Parse command line arguments
+        1. Server URL
+        2. Number of total users
+            - default: 10 users
+        3. Number of parallel requests 
+            - default: 10 users hitting the server at the same time.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--server_url", 
+        type=str, 
+        default="https://tabpfn-server-wjedmz7r5a-ez.a.run.app",
+        help="Server URL"
+    )
+    parser.add_argument(
+        "--num_users", 
+        type=int, 
+        default=10,
+        help="Number of Users"
+    )
+    parser.add_argument(
+        "--num_requests", 
+        type=int, 
+        default=10,
+        help="Number of parallel requests"
+    )
+    return parser.parse_args()
+
+def generate_email():
+    """
+    Generate a random email address for a user: 
+    """
+    random_uuid = uuid.uuid4().hex.replace('%', '')
```

Comment:
add a comment, how many email addresses are possible here to give a rough idea what percentage of collisions are to be expected

Reply:
Hi Sam. This can generate around 16^32 unique email addresses as of now and can be increased by increasing the length of the UUID. So the probability of generating the same email address for the second time is negligibly low. 
Approximately, there would be 1 - (16^32! / (16^32^100 * (16^32 - 100)!)) collisions using the probability birthday theorem

Reply:
thanks for adding this!